corehq/apps/geospatial/urls.py

```diff
@@ -1,19 +1,22 @@
+# urls.py
```

Comment:
You didn't like the file name Norman :)

Reply:
:rofl: That means I copied-and-pasted from notes in my text editor. :roll_eyes: I'll remove it.